code/clue.js

```diff
@@ -10,50 +9,202 @@ const mrGreen = {
   occupation: 'Entrepreneur'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is very quiet',
+  age: 39,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is very smart',
+  age: 37,
+  image: 'assets/scarlet.png',
+  occupation: 'Lawyer'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is very rich',
+  age: 58,
+  image: 'assets/peacock.png',
+  occupation: 'Doctor'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is very loyal',
+  age: 62,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She is very kind',
+  age: 65,
+  image: 'assets/white.png',
+  occupation: 'Actress'
+}
 
 const rope = {
   name: 'rope',
-  weight: 10
+  weight: 10,
+  color: 'black',
+  image: 'assets/rope.png'
+}
+
+const knife = {
+  name: 'knife',
+  weight: 4,
+  color: 'black',
+  image: 'assets/knife.png'
+}
+
+const candlestick = {
+  name: 'candestick',
+  weight: 8,
+  color: 'black',
+  image: 'assets/candlestick.png',
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 14,
+  color: 'black',
+  image: 'assets/dumbbell.png'
+}
+
+const poison = {
+  name: 'poison',
+  weight: 1,
+  color: 'black',
+  image: 'assets/poison.png'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const axe = {
+  name: 'axe',
+  weight: 12,
+  color: 'black',
+  image: 'assets/axe.png'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const bat = {
+  name: 'bat',
+  weight: 9,
+  color: 'black',
+  image: 'assets/bat.png'
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 7,
+  color: 'black',
+  image: 'assets/trophy.png'
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 5,
+  color: 'black',
+  image: 'assets/pistol.png'
+}
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  mrsPeacock,
+  colonelMustard,
+  professorPlum,
+  missScarlet
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
 ]
 
-const weapons = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
+]
+
+console.log(suspects)
+console.log(weapons)
+console.log(rooms)
```

Comment:
Great that you were console logging your objects to see if they're there, better to do it early on in the projects, so props for that! 🤗 Remember to delete your console logs once you're 'done' with them. They don't need to be in your final commits/final versions of files.